tests/test_signature_json.py

```diff
@@ -5,13 +5,15 @@
 import sourmash
 from sourmash.signature import SourmashSignature
 from sourmash.signature_json import (_json_next_atomic_array,
-                                         _json_next_signature,
-                                         load_signature_json,
-                                         load_signatures_json,
-                                         load_signatureset_json_iter,
-                                         save_signatures_json)
+                                     _json_next_signature,
```

Comment:
please revert spacing changes, so we can see what actually was changed semantically - thx!

Reply:
reverted